src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,129 @@
+"""The following are the unpolarized time-like leading-order (LO) Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from eko import constants
+
+
+
+@nb.njit(cache=True)
+def gamma_qq(N, s1):
+    """Computes the LO quark-quark anomalous dimension.
+    Implements Eqn. (B.3) from hep-ph/0604160.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment 
+    s1 : complex
+        Harmonic sum $S_{1}$ 
+
+    Returns
+    -------
+    gamma_qq : complex
+        LO quark-quark anomalous dimension $\gamma_{qq}^{(0)}(N)$ 
+
+    """
+    return constants.CF * (-3.0 + (4.0 * s1) - 2.0 / (N * (N + 1.0)))
+
+@nb.njit(cache=True)
+def gamma_qg(N):
+    """Computes the LO quark-gluon anomalous dimension.
+    Implements Eqn. (B.4) from hep-ph/0604160 and Eqn. (A1) from PhysRevD.48.116.
```

Comment:
Add the references as proper entries here https://github.com/NNPDF/eko/blob/7a74ddbfdfff5b485de3aec20d7f1fbcd33b6c29/doc/source/refs.bib and cite them using sphinx

Reply:
Actually, since this function will be a redirect (see other comment) I wonder whether you can add a docstring to a variable, but I think you can ...

Reply:
> but I think you can ...

For top-level variables it is supported by Sphinx (maybe even by Python itself, but this I should test).

Reply:
> Add the references as proper entries here https://github.com/NNPDF/eko/blob/7a74ddbfdfff5b485de3aec20d7f1fbcd33b6c29/doc/source/refs.bib and cite them using sphinx

thanks for pointing this out. Will do so in the next few commits